src/components/WelcomeForm/WelcomeForm.jsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./WelcomeForm.module.css";
+import Message from "./Message";
+import { useState } from "react";
+
+const WelcomeForm = () => {
+  const [user, setUser] = useState({ userName: "", age: "" });
```

Comment:
Here I fully encourage you to change the type of `age` to a number type and initially a `null` value ie: `({ userName: '', age: null })`